### ISSUE TITLE
build: Android NDK version mismatch

### ISSUE THREAD

**Issue by carun** [zulip/zulip-flutter#1220]
```
Your project is configured with Android NDK 26.1.10909125, but the following plugin(s) depend on a different Android NDK version:
- integration_test requires Android NDK 26.3.11579264
Fix this issue by using the highest Android NDK version (they are backward compatible).
Add the following to C:\src\zulip-flutter\android\app\build.gradle:

    android {
        ndkVersion = "26.3.11579264"
        ...
    }
```

**Comment by chrisbobbe** [comment]
Thanks for the report; helpful to have an issue tracking this. We're discussing it in the [Zulip development community](https://zulip.com/development-community/), [here](https://chat.zulip.org/#narrow/channel/516-mobile-dev-help/topic/Android.20NDK.20version/near/2017496).

**Comment by gnprice** [comment]
To make it explicit on this thread: that message is only a warning. So even before we fix it, it shouldn't get in the way of building and running the app.

**Comment by carun** [comment]
@chrisbobbe thanks. I'm trying to login using my Gmail, however it shows an error. Not sure who to reach out to. 

> Your account visionofarun@gmail.com has been deactivated. Please contact your organization administrator to reactivate it.

**Comment by chrisbobbe** [comment]
Do you mean you're trying to log into chat.zulip.org? Or is it a different Zulip organization?

**Comment by carun** [comment]
I'm trying to log into chat.zulip.org.

**Comment by gnprice** [comment]
For contacting the administrators of chat.zulip.org, you can use support@zulip.com; I think we may not have a more specific address to use.

Note that replies this week and next may be slow due to holidays.

**Comment by drrnbrns** [comment]
I'm not sure if this is the same issue (let me know if I should log a separate bug), but with 3.29.0 (stable) I'm getting this warning.  Either the mismatch detection is new, or Flutter actually downgraded to an [unsupported version of the NDK](https://github.com/android/ndk/wiki/Unsupported-Downloads).  Is that expected?

I know it's just a warning and there's a workaround, but it doesn't seem right that a new project should use a deprecated NDK.
```
Your project is configured with Android NDK 26.3.11579264, but the following plugin(s) depend on a different Android NDK version:
- audioplayers_android requires Android NDK 27.0.12077973
- path_provider_android requires Android NDK 27.0.12077973
- sensors_plus requires Android NDK 27.0.12077973
```

This can be reproduced easily with 

```
flutter create --platforms=android ndk_sample
cd ndk_sample
flutter pub add audioplayers
flutter run
````

**Comment by gnprice** [comment]
> it doesn't seem right that a new project should use a deprecated NDK.

Yeah, I agree. Sounds like a good issue to report in the Flutter issue tracker 🙂, flutter/flutter.

This thread is in the tracker of a particular Flutter app, namely Zulip.

**Comment by Sesa1988** [comment]
is there something new? I also just upgraded Android Studio and AGP and get this error. I changed the following:

//ndkVersion flutter.ndkVersion
ndkVersion = "27.0.12077973"

but it feels like a workaround since it referencing something from Flutter here that should match?

**Comment by gnprice** [comment]
@Sesa1988 It looks like you subsequently found the issue thread for this in Flutter's own tracker 🙂: https://github.com/flutter/flutter/issues/139427

This thread is in the issue tracker for Zulip, a particular Flutter app.

(For cross-reference: it looks like after the earlier comments above, @drrnbrns filed https://github.com/flutter/flutter/issues/163945 in the upstream tracker, and that was consolidated into the other issue.)

For anyone else who comes across this thread: that message is a warning, not an error. As far as we've seen in Zulip to date, it doesn't get in the way of the app building and running just fine. So although it'd be good to clear up (hence this issue thread and the upstream one), one can also just ignore it until the upstream issue gets fixed.
